climada/hazard/tc_tracks_synth.py

```diff
@@ -71,17 +80,38 @@
 )
 v_rel, p_rel = _calc_land_decay(tracks.data, land_geom, pool=tracks.pool)"""
 
-def calc_perturbed_trajectories(tracks,
-                                nb_synth_tracks=9,
-                                max_shift_ini=0.75,
-                                max_dspeed_rel=0.3,
-                                max_ddirection=np.pi / 360,
-                                autocorr_dspeed=0.85,
-                                autocorr_ddirection=0.5,
-                                seed=CONFIG.hazard.trop_cyclone.random_seed.int(),
-                                decay=True,
-                                use_global_decay_params=True,
-                                pool=None):
+DATA_DIR = Path(__file__).parent.joinpath('data')
+RANDOM_WALK_DATA_INTENSIFICATION = pd.read_csv(DATA_DIR.joinpath('tc_intensification_params.csv'))
+RANDOM_WALK_DATA_INTENSIFICATION['cumfreqlow'] = np.append(-0.01, RANDOM_WALK_DATA_INTENSIFICATION['freq'].values.cumsum()[:-1])
+RANDOM_WALK_DATA_INTENSIFICATION['cumfreqhigh'] = np.append(RANDOM_WALK_DATA_INTENSIFICATION['cumfreqlow'].values[1:], 1.0)
+RANDOM_WALK_DATA_DURATION = pd.read_csv(DATA_DIR.joinpath('tc_peak_params.csv'), na_values='', keep_default_na=False)
+RANDOM_WALK_DATA_SEA_DECAY = pd.read_csv(DATA_DIR.joinpath('tc_decay_params.csv'), na_values='', keep_default_na=False)
+
+RANDOM_WALK_DATA_CAT_STR = {
+    -1: 'TD-TS',
+    0: 'TD-TS',
+    1: "Cat 1",
+    2: "Cat 2",
+    3: "Cat 3",
+    4: "Cat 4",
+    5: "Cat 5"
+}
+
+def calc_perturbed_trajectories(
+    tracks,
+    nb_synth_tracks: int = 9,
+    max_shift_ini: float = 0.75,
+    max_dspeed_rel: float = 0.3,
+    max_ddirection: float = np.pi / 360,
+    autocorr_dspeed: float = 0.85,
+    autocorr_ddirection: float = 0.5,
+    seed: int = CONFIG.hazard.trop_cyclone.random_seed.int(),
+    adjust_intensity: bool = True,
+    central_pressure_pert: float = 7.5,
+    legacy_decay: bool = False,
```

Comment:
Why did you rename `decay` to `legacy_decay`? The way I has implemented it in my first commits is that `decay` was a legacy parameter, however if a user set it they get a warning to use `adjust_intensity` instead and that parameter value is overwritten by `decay`.